### ISSUE TITLE
Qt: IPFilter do not work with specific IP addresses

### ISSUE THREAD

**Issue by ValdikSS** [eiskaltdcpp/eiskaltdcpp#378]
Enabling IPFilter with added IP address blocked in any direction does nothing, the user is still able to download shared files from EiskaltDC++.
I'm using current git master.

**Comment by tehnick** [comment]
Which version of EiskaltDC++? Which type of UI? Example of config with list of blocked IP addresses.

**Comment by ValdikSS** [comment]
Latest git master, Qt UI. I'm trying to add individual IP address using the interface, not the IP list.

**Comment by tehnick** [comment]
Show the example of string please.

**Comment by ValdikSS** [comment]
![screenshot_20180830_231035](https://user-images.githubusercontent.com/3054729/44876474-e79bd480-aca9-11e8-8308-44cc2f200455.png)

**Comment by tehnick** [comment]
It is interesting: after a quick glance it looks like we have two separate IP filters: one in Qt UI and one in DC++ Core.

**Comment by ValdikSS** [comment]
@tehnick, Qt interface hang in 5 seconds after entering this command, and EiskaltDC has crashed.

**Comment by tehnick** [comment]
It is very strange! In my system all works fine...

In any case this command is just for changing DC++ core setting in runtime. Your may quit from program and set this option manually in `~/.config/eiskaltdc++/DCPlusPlus.xml`. And then launch program again.

**Comment by tehnick** [comment]
As far I see IP filter config should be in `~/.config/eiskaltdc++/ipfilter`.

**Comment by ValdikSS** [comment]
I've added the following parameter into DCPlusPlus.xml:
```
<DCPlusPlus>
        <Settings>
                <IpFilter type="int">1</IpFilter>
```
But the filter configured in Qt interface doesn't seem to work.

**Comment by SeanTryGit** [comment]
ValdikSS is totatly Right there is a problem with IP Filter with Eiskaltdcc++ v2.2.10 in linux, it doesn't work.

**Comment by SeanTryGit** [comment]
is there someone who can do something ? because this setting is really really important.... :)

**Comment by tehnick** [comment]
Try current `master` please.

@MKras have replaced extra implementation of IP filter specific for Qt by one from a Core.

**Comment by SeanTryGit** [comment]
ok i will check it from my side to see if it's working or not

**Comment by tehnick** [comment]
It should be fixed in version `2.4.1` or since commit 651fe82125117f33cfb0f8dbfdf4e8fe18b09e64 for those who use builds from master branch.
